# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1472235186 -7200
#      Fri Aug 26 20:13:06 2016 +0200
# Node ID 5b9f2502b1dd098bf83992fbda0fc65077df9611
# Parent  52c205a0ad888552a7c62c23e8c47495679edafe
Update French translation for 4.2 release

* fr_FR.ts: add missing translations
* fr_FR.ts: spell check all translations
* fr_FR.ts: consistent spelling of "bogue/déboguage..." (bug...) which have no
  stable spelling in french ("débugger, débuguer ...")

diff --git a/libgui/languages/fr_FR.ts b/libgui/languages/fr_FR.ts
--- a/libgui/languages/fr_FR.ts
+++ b/libgui/languages/fr_FR.ts
@@ -59,40 +59,40 @@
     <message>
         <location line="+1"/>
         <source>cursor</source>
         <translation>curseur</translation>
     </message>
     <message>
         <location filename="../src/settings-dialog.cc" line="+132"/>
         <source>Difference to the default size</source>
-        <translation type="unfinished">Différence avec la taille par défaut</translation>
+        <translation>Différence avec la taille par défaut</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255,0,255) means default</source>
-        <translation type="unfinished">Couleur de fond, rose [255,0,255] designe la valeur par défaut</translation>
+        <translation>Couleur de fond, magenta [255,0,255] désigne la valeur par défaut</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>b</source>
         <comment>short form for bold</comment>
-        <translation type="unfinished">g</translation>
+        <translation>g</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>i</source>
         <comment>short form for italic</comment>
-        <translation type="unfinished">i</translation>
+        <translation>i</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>u</source>
         <comment>short form for underlined</comment>
-        <translation type="unfinished">s</translation>
+        <translation>s</translation>
     </message>
 </context>
 <context>
     <name>QTerminal</name>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.h" line="+130"/>
         <source>Copy</source>
         <translation>Copier</translation>
@@ -110,25 +110,25 @@
     <message>
         <location line="+7"/>
         <source>Clear Window</source>
         <translation>Nettoyer la Fenêtre de Commande</translation>
     </message>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.cc" line="+46"/>
         <source>Edit %1 at line %2</source>
-        <translation type="unfinished"></translation>
+        <translation>Éditer %1 à la ligne %2</translation>
     </message>
 </context>
 <context>
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1779"/>
         <source>copied selection to clipboard</source>
-        <translation>sélection copiée vers le presse-papier</translation>
+        <translation>sélection copiée vers le presse-papiers</translation>
     </message>
 </context>
 <context>
     <name>QtHandles::Figure</name>
     <message>
         <location filename="../graphics/Figure.cc" line="+298"/>
         <source>Figure ToolBar</source>
         <translatorcomment>This may be long depending on the context.</translatorcomment>
@@ -178,17 +178,17 @@
         <location line="+2"/>
         <source>Cop&amp;y</source>
         <translation>&amp;Copier</translation>
     </message>
     <message>
         <location line="+7"/>
         <location line="+463"/>
         <source>About Octave</source>
-        <translation type="unfinished">À propos d&apos;Octave</translation>
+        <translation>À propos d&apos;Octave</translation>
     </message>
     <message>
         <location line="-465"/>
         <source>&amp;Help</source>
         <translation>&amp;Aide</translation>
     </message>
     <message>
         <location line="+504"/>
@@ -213,181 +213,188 @@
     <message>
         <location line="+4"/>
         <source>Zoom Out</source>
         <translation>Réduire</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Pan</source>
-        <translation>Deplacer</translation>
+        <translation>Déplacer</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Insert Text</source>
-        <translation>Inserer du Texte</translation>
+        <translation>Insérer du Texte</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Select</source>
         <translation>Sélectionner</translation>
     </message>
 </context>
 <context>
     <name>annotation_dialog</name>
     <message>
         <location filename="../graphics/annotation-dialog.ui" line="+17"/>
         <source>Annotation</source>
-        <translation type="unfinished"></translation>
+        <translation>Annotation</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Text</source>
-        <translation type="unfinished"></translation>
+        <translation>Texte</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>String</source>
-        <translation type="unfinished"></translation>
+        <translation>Texte</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Font</source>
-        <translation type="unfinished">Police</translation>
+        <translation>Police</translation>
     </message>
     <message>
         <location line="+42"/>
         <source>bold</source>
-        <translation type="unfinished"></translation>
+        <translation>gras</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>b</source>
-        <translation type="unfinished">g</translation>
+        <translation>g</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>italic</source>
-        <translation type="unfinished"></translation>
+        <translation>italique</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>i</source>
-        <translation type="unfinished">i</translation>
+        <translation>i</translation>
     </message>
     <message>
         <location line="+428"/>
         <source>color</source>
-        <translation type="unfinished"></translation>
+        <translation>couleur</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>Horz alignment</source>
-        <translation type="unfinished"></translation>
+        <translation>Alig. Horizontal</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>left</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Do not translate!</translatorcomment>
+        <translation>left</translation>
     </message>
     <message>
         <location line="+5"/>
         <location line="+29"/>
         <source>middle</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Do not translate</translatorcomment>
+        <translation>middle</translation>
     </message>
     <message>
         <location line="-24"/>
         <source>right</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Do not translate</translatorcomment>
+        <translation>right</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Vert alignment</source>
-        <translation type="unfinished"></translation>
+        <translation>Alig. Vertical</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>top</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Do not translate</translatorcomment>
+        <translation>top</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>bottom</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Do not translate</translatorcomment>
+        <translation>bottom</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Position</source>
-        <translation type="unfinished"></translation>
+        <translation>Position</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>X</source>
-        <translation type="unfinished"></translation>
+        <translation>X</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Y</source>
-        <translation type="unfinished"></translation>
+        <translation>Y</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Width</source>
-        <translation type="unfinished"></translation>
+        <translation>Largeur</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Height</source>
-        <translation type="unfinished"></translation>
+        <translation>Hauteur</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>units</source>
-        <translation type="unfinished"></translation>
+        <translation>unités</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>normalized</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Do not translate</translatorcomment>
+        <translation>normalized</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Fit to box</source>
-        <translation type="unfinished"></translation>
+        <translation>Adapter au texte</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Box</source>
-        <translation type="unfinished"></translation>
+        <translation>Boîte</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background</source>
-        <translation type="unfinished"></translation>
+        <translation>Arrière plan</translation>
     </message>
     <message>
         <location line="+444"/>
         <source>Edge</source>
-        <translation type="unfinished"></translation>
+        <translation>Bord</translation>
     </message>
     <message>
         <location line="+133"/>
         <source>Line style</source>
-        <translation type="unfinished"></translation>
+        <translation>Style de ligne</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>none</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Do not translate</translatorcomment>
+        <translation>none</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Line width</source>
-        <translation type="unfinished"></translation>
+        <translation>Épaisseur de ligne</translation>
     </message>
 </context>
 <context>
     <name>documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+34"/>
         <source>Documentation</source>
         <translation>Documentation</translation>
@@ -427,17 +434,17 @@
 </source>
         <translation>Nom de la nouvelle fonction :</translation>
     </message>
     <message>
         <location line="+395"/>
         <source>File not saved! A file with the selected name
 %1
 is already open in the editor</source>
-        <translation>Fichier non-enregistré! Un fichier ayant le nom sélectionné
+        <translation>Fichier non enregistré! Un fichier ayant le nom sélectionné
 %1
 est déjà ouvert dans l&apos;éditeur</translation>
     </message>
     <message>
         <location line="+492"/>
         <source>&amp;%1 %2</source>
         <translation>&amp;%1 %2</translation>
     </message>
@@ -586,17 +593,17 @@ est déjà ouvert dans l&apos;éditeur</translation>
     <message>
         <location line="+32"/>
         <source>&amp;Commands</source>
         <translation>&amp;Commandes</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Delete Line</source>
-        <translation></translation>
+        <translation>Effacer la ligne</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Copy Line</source>
         <translation>Copier la Ligne</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -706,67 +713,67 @@ est déjà ouvert dans l&apos;éditeur</translation>
     <message>
         <location line="+3"/>
         <source>Show &amp;Line Numbers</source>
         <translation>Afficher les Numéros de &amp;Lignes</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>Show &amp;Toolbar</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher la barre  d&apos;Ou&amp;tils</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show &amp;Statusbar</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher la barre de &amp;statut</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show &amp;Horizontal Scrollbar</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher la barre de navigation &amp;horizontale</translation>
     </message>
     <message>
         <location line="+69"/>
         <source>Recent Files</source>
-        <translation type="unfinished"></translation>
+        <translation>Fichiers récents</translation>
     </message>
     <message>
         <location line="+65"/>
         <source>Editor</source>
         <translation>Éditeur</translation>
     </message>
     <message>
         <location line="-156"/>
         <source>Show Line &amp;Endings</source>
         <translation>Afficher les Fins de Lign&amp;es</translation>
     </message>
     <message>
         <location line="-1467"/>
         <source>File Encoding:</source>
-        <translation type="unfinished"></translation>
+        <translation>Encodage :</translation>
     </message>
     <message>
         <location line="+403"/>
         <source>%1 is a built-in function</source>
-        <translation type="unfinished">%1 est une fonction interne</translation>
+        <translation>%1 est une fonction interne</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Can not find function %1</source>
-        <translation type="unfinished">Impossible de trouver la fonction %1</translation>
+        <translation>Impossible de trouver la fonction %1</translation>
     </message>
     <message>
         <location line="+911"/>
         <source>Find &amp;Next...</source>
-        <translation type="unfinished"></translation>
+        <translation>Trouver le Suiva&amp;nt...</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Find &amp;Previous...</source>
-        <translation type="unfinished"></translation>
+        <translation>Trouver le &amp;Précédent...</translation>
     </message>
     <message>
         <location line="+118"/>
         <source>Show &amp;Indentation Guides</source>
         <translation>Afficher les Guides d&apos;&amp;Indentation</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -786,17 +793,17 @@ est déjà ouvert dans l&apos;éditeur</translation>
     <message>
         <location line="+2"/>
         <source>&amp;Normal Size</source>
         <translation>Taille &amp;Normale</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Run &amp;Selection</source>
-        <translation>Exécuter la &amp;Selection</translation>
+        <translation>Exécuter la &amp;Sélection</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
         <translation>A&amp;ide</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -819,17 +826,17 @@ pour lecture : %2.</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
         <translation>Le fichier
 %1
-n&apos;existe pas. Voulez-vous le créer?</translation>
+n&apos;existe pas. Voulez vous le créer?</translation>
     </message>
     <message>
         <location line="+25"/>
         <source>Could not open file
 %1
 for write: %2.</source>
         <translation>Impossible d&apos;ouvrir le fichier
 %1 
@@ -853,17 +860,17 @@ pour écrire : %2.</translation>
     <message>
         <location line="+129"/>
         <source>Show &amp;Whitespace Characters</source>
         <translation>Afficher les e&amp;spaces</translation>
     </message>
     <message>
         <location line="+42"/>
         <source>&amp;Debug</source>
-        <translation>&amp;Débuguer</translation>
+        <translation>&amp;Déboguer</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>&amp;Run</source>
         <translation>E&amp;xécuter</translation>
     </message>
 </context>
 <context>
@@ -910,95 +917,107 @@ is about to be closed but has been modif
 
   %1
 
 est sur le point d&apos;être fermé mais a été modifié.  %2</translation>
     </message>
     <message>
         <location line="+298"/>
         <source>Debug or Save</source>
-        <translation type="unfinished"></translation>
+        <translation>Déboguer ou Enregistrer</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>This file is currently being executed.
 Quit debugging and save?</source>
-        <translation type="unfinished"></translation>
+        <translation>Ce fichier est en cours d&apos;exécution.
+Quitter le mode déboguage et sauver?</translation>
     </message>
     <message>
         <location line="+166"/>
         <source>Line Endings:</source>
         <translation>Fins de Lignes:</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>File Encoding:</source>
-        <translation type="unfinished"></translation>
+        <translation>Encodage :</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>Octave Files (*.m)</source>
         <translation>Fichiers Octave (*.m)</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>All Files (*)</source>
         <translation>Tous les Fichiers (*)</translation>
     </message>
     <message>
         <location line="-513"/>
         <source>Do you want to cancel closing, save or discard the changes?</source>
-        <translation>Voulez-vous annuler la fermeture, enregistrer ou supprimer les modifications ?</translation>
+        <translation>Voulez vous annuler la fermeture, enregistrer ou supprimer les modifications ?</translation>
     </message>
     <message>
         <location line="-1342"/>
         <source>encoding:</source>
-        <translation type="unfinished"></translation>
+        <translation>encodage :</translation>
     </message>
     <message>
         <location line="+239"/>
         <source>Breakpoint condition</source>
-        <translation type="unfinished"></translation>
+        <translation>Condition de point d&apos;arrêt</translation>
     </message>
     <message>
         <location line="+88"/>
         <location line="+1024"/>
         <location line="+372"/>
         <location line="+215"/>
         <location line="+21"/>
         <location line="+83"/>
         <location line="+23"/>
         <source>Octave Editor</source>
         <translation>Éditeur d&apos;Octave</translation>
     </message>
     <message>
         <location line="-1737"/>
         <source>Cannot add breakpoint to modified file.
 Save and add breakpoint, or canel?</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossible d&apos;ajouter un point d&apos;arrêt à un fichier modifié.
+Sauvegarder et ajouter le point d&apos;arrêt ou annuler?</translation>
     </message>
     <message>
         <location line="+1611"/>
         <source>&quot;%1&quot;
 is not a valid identifier.
 
 If you keep this filename, you will not be able to
 call your script using its name as an Octave command.
 
 Do you want to choose another name?</source>
-        <translation type="unfinished"></translation>
+        <translation>&quot;%1&quot;
+n&apos;est pas un nom valide.
+
+Si vous gardez ce nom de fichier, vous ne pourrez pas exécuter
+votre script en utilisant son nom en guise de commande Octave.
+
+Voulez vous choisir un autre nom?</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>The current editor contents can not be encoded
 with the selected codec %1.
 Using it will result in data loss!
 
 Do you want to chose another codec?</source>
-        <translation type="unfinished"></translation>
+        <translation>Le contenu actuel de l&apos;éditeur ne peut être encodé
+avec le codec sélectionné %1.
+Cela mènerait à des pertes de données!
+
+Voulez vous choisir un autre codec?</translation>
     </message>
     <message>
         <location line="+99"/>
         <source>
 
 Warning: The contents in the editor is modified!</source>
         <translation>
 
@@ -1006,29 +1025,29 @@ Avertissement: Le contenu dans l&apos;éditeur est modifié!</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>It seems that the file
 %1
 has been deleted or renamed. Do you want to save it now?%2</source>
         <translation>Il semblerait que le fichier
 %1
-a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
+a été supprimé ou renommé. Voulez vous l&apos;enregistrer maintenant ?%2</translation>
     </message>
     <message>
         <location line="-342"/>
         <source>Could not open file %1 for write:
 %2.</source>
         <translation>Impossible d&apos;ouvrir le fichier %1 pour écrire :
 %2.</translation>
     </message>
     <message>
         <location line="+319"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
-        <translation>Il semblerait que &apos;%1&apos; a été modifié par une autre application. Voulez-vous le récharger ?</translation>
+        <translation>Il semblerait que &apos;%1&apos; a été modifié par une autre application. Voulez vous le recharger ?</translation>
     </message>
 </context>
 <context>
     <name>files_dock_widget</name>
     <message>
         <location filename="../src/files-dock-widget.cc" line="+69"/>
         <source>File Browser</source>
         <translation>Explorateur de fichiers</translation>
@@ -1057,67 +1076,72 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <location line="+4"/>
         <location line="+373"/>
         <source>Find Files...</source>
         <translation>Rechercher des fichiers...</translation>
     </message>
     <message>
         <location line="-26"/>
         <source>Open in System File Explorer</source>
-        <translation type="unfinished"></translation>
+        <translation>Ouvrir dans l&apos;explorateur de fichiers du système</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Open in Text Editor</source>
-        <translation type="unfinished"></translation>
+        <translation>Ouvrir dans l&apos;éditeur de texte</translation>
     </message>
     <message>
         <location line="+389"/>
         <source>New Directory</source>
         <translation>Nouveau répertoire</translation>
     </message>
     <message>
         <location line="-773"/>
         <source>Show Octave directory</source>
         <translation>Revenir au répertoire d&apos;Octave</translation>
     </message>
     <message>
         <location line="-49"/>
         <source>Browse your files</source>
-        <translation type="unfinished"></translation>
+        <translation>Examinez vos fichiers</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Alternating row colors</source>
-        <translation type="unfinished">Alterner les couleurs des lignes</translation>
+        <translation>Alterner les couleurs des lignes</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>filesdockwidget/showFileSize</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Not translated</translatorcomment>
+        <translation>filesdockwidget/showFileSize</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>filesdockwidget/showFileType</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Not translated</translatorcomment>
+        <translation>filesdockwidget/showFileType</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>filesdockwidget/showLastModified</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Not translated</translatorcomment>
+        <translation>filesdockwidget/showLastModified</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>filesdockwidget/showHiddenFiles</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Not translated</translatorcomment>
+        <translation>filesdockwidget/showHiddenFiles</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>filesdockwidget/useAlternatingRowColors</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Not translated</translatorcomment>
+        <translation>filesdockwidget/useAlternatingRowColors</translation>
     </message>
     <message>
         <location line="+29"/>
         <source>One directory up</source>
         <translation>Monter au répertoire parent</translation>
     </message>
     <message>
         <location line="+6"/>
@@ -1144,17 +1168,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <location line="-379"/>
         <location line="+382"/>
         <source>New Directory...</source>
         <translation>Nouveau répertoire...</translation>
     </message>
     <message>
         <location line="-329"/>
         <source>Activate to open in editor, right click for alternatives</source>
-        <translation>Activer pour ouvrir dand l&apos;éditeur, click droit pour les alternatives</translation>
+        <translation>Activer pour ouvrir dans l&apos;éditeur, click droit pour les alternatives</translation>
     </message>
     <message>
         <location line="-136"/>
         <source>File size</source>
         <translation>Taille du fichier</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1204,23 +1228,23 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location line="+2"/>
         <source>Delete...</source>
         <translation>Effacer...</translation>
     </message>
     <message>
         <location line="+125"/>
         <source>Rename file/directory</source>
-        <translation>Rénommer fichier/répertoire</translation>
+        <translation>Renommer fichier/répertoire</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Rename file/directory:
 </source>
-        <translation>Rénommer fichier/répertoire:
+        <translation>Renommer fichier/répertoire:
 </translation>
     </message>
     <message>
         <location line="+1"/>
         <source>
  to: </source>
         <translation>
  en : </translation>
@@ -1306,17 +1330,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;We hope you find Octave to be a useful tool.&lt;/p&gt;
 &lt;p&gt;If you encounter problems, there are a number of ways to get help, including commercial support options, a mailing list, a wiki, and other community-based support channels.
 You can find more information about each of these by visiting &lt;a href=&quot;http://octave.org/support.html&quot;&gt;http://octave.org/support.html&lt;/a&gt; (opens in external browser).&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;Nous espérons que vous trouvez Octave utile.&lt;/p&gt;
-&lt;p&gt;Si vous rencontrez des difficultés, il y a plusieurs façon de trouver de l&apos;aide parmis les-quelles des options de support commercial, une liste mail, un wiki et d&apos;autres ressources communautaires.
+&lt;p&gt;Si vous rencontrez des difficultés, il y a plusieurs façon de trouver de l&apos;aide parmi lesquelles des options de support commercial, une liste mail, un wiki et d&apos;autres ressources communautaires.
 Vous trouverez des information sur chacune d&apos;entre elles sur &lt;a href=&quot;http://octave.org/support.html&quot;&gt;http://octave.org/support.html&lt;/a&gt; (ouvert dans un navigateur externe).&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
@@ -1461,37 +1485,37 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+10"/>
         <source>Named:</source>
         <translation>Nommé:</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Start in:</source>
-        <translation>Demarrer dans :</translation>
+        <translation>Démarrer dans :</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Enter the start directory</source>
-        <translation>Répetoire de démarrage</translation>
+        <translation>Répertoire de démarrage</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Browse...</source>
         <translation>Naviguer...</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Browse for start directory</source>
         <translation>Choisir le répertoire de démarrage</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Search recursively through directories for matching files</source>
-        <translation>Rechercher les fichiers recursivement dans les sous-répertoires</translation>
+        <translation>Rechercher les fichiers récursivement dans les sous-répertoires</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Include matching directories in search results</source>
         <translation>Inclure les répertoires concordants dans des résultats de la recherche</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -1512,17 +1536,17 @@ a:link { text-decoration: underline; col
         <location line="-35"/>
         <source>Enter the filename search expression</source>
         <translatorcomment>Not sure I understand the </translatorcomment>
         <translation>Entrer l&apos;expression de recherche de nom de fichier</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Search subdirectories</source>
-        <translation>Rechercher dans les sous-dossiers</translation>
+        <translation>Rechercher dans les sous dossiers</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Include directory names</source>
         <translation>Inclure les noms de dossiers</translation>
     </message>
     <message>
         <location line="+12"/>
@@ -1572,32 +1596,32 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Stop searching</source>
         <translation>Arrêter la recherche</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Filename/location</source>
-        <translation type="unfinished"></translation>
+        <translation>Nom/Emplacement du fichier</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>File contents</source>
         <translation>Contenu du fichier</translation>
     </message>
     <message>
         <location line="+111"/>
         <source>Searching...</source>
         <translation>Recherche en cours...</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>%1 match(es)</source>
-        <translation></translation>
+        <translation>%1 resultat(s)</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Set search directory</source>
         <translation>Définir le répertoire de recherche</translation>
     </message>
 </context>
 <context>
@@ -1618,17 +1642,17 @@ a:link { text-decoration: underline; col
     <message>
         <location filename="../src/history-dock-widget.cc" line="+47"/>
         <source>Browse and search the command history.</source>
         <translation>Naviguer et rechercher l&apos;historique des commandes.</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Double-click a command to transfer it to the terminal.</source>
-        <translation>Double cliquez sur une commande pour la transferer au terminal.</translation>
+        <translation>Double cliquez sur une commande pour la transférer au terminal.</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Enter text to filter the command history</source>
         <translation>Entrer du texte pour filtrer l&apos;historique des commandes</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -1653,22 +1677,22 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+3"/>
         <source>Create script</source>
         <translation>Créer un fichier de script</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Hide filter</source>
-        <translation type="unfinished"></translation>
+        <translation>Cacher le filtre</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show filter</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher le filtre</translation>
     </message>
 </context>
 <context>
     <name>initial_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="-176"/>
         <source>Welcome to Octave!</source>
         <translation>Bienvenue dans Octave !</translation>
@@ -1686,17 +1710,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;You seem to be using the Octave graphical interface for the first time on this computer.
 Click &apos;Next&apos; to create a configuration file and launch Octave.&lt;/p&gt;
 &lt;p&gt;The configuration file is stored in&lt;br&gt;%1.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;body&gt;
-&lt;p&gt;Vous semblez utiliser l&apos;interface graphique d&apos;Octave pour la première fois sur cet ordianteur.
+&lt;p&gt;Vous semblez utiliser l&apos;interface graphique d&apos;Octave pour la première fois sur cet ordinateur.
 Cliquez &quot;Suivant&quot; pour créer un fichier de configuration et lancer Octave.&lt;/p&gt;
 &lt;p&gt;Le fichier de configuration est enregistré dans&lt;br&gt;%1.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
@@ -1828,17 +1852,17 @@ Cliquez &quot;Suivant&quot; pour créer un fichier de configuration et lancer Octave.&lt;/p&gt;
     <message>
         <location line="+3"/>
         <source>Clear Workspace</source>
         <translation>Nettoyer l&apos;espace de travail</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>De&amp;bug</source>
-        <translation>Dé&amp;buguer</translation>
+        <translation>Dé&amp;boguer</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Step</source>
         <translation>Avancer d&apos;un pas</translation>
     </message>
     <message>
         <location line="+12"/>
@@ -1853,22 +1877,22 @@ Cliquez &quot;Suivant&quot; pour créer un fichier de configuration et lancer Octave.&lt;/p&gt;
     <message>
         <location line="+20"/>
         <source>File Browser</source>
         <translation>Explorateur de fichiers</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Contribute</source>
-        <translation type="unfinished"></translation>
+        <translation>Contribuer</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Donate to Octave</source>
-        <translation type="unfinished"></translation>
+        <translation>Donner à Octave</translation>
     </message>
     <message>
         <location line="-123"/>
         <source>Step In</source>
         <translation>Avancer d&apos;un pas avec entrée</translation>
     </message>
     <message>
         <location line="-1082"/>
@@ -1959,17 +1983,17 @@ Cliquez &quot;Suivant&quot; pour créer un fichier de configuration et lancer Octave.&lt;/p&gt;
         <location line="-1198"/>
         <location line="+1201"/>
         <source>Browse directories</source>
         <translation>Naviguer dans les répertoires</translation>
     </message>
     <message>
         <location line="-175"/>
         <source>Quit Debug Mode</source>
-        <translation>Quitter le mode Débuguage</translation>
+        <translation>Quitter le mode Déboguage</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>&amp;Window</source>
         <translation>Fenê&amp;tre</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -2025,17 +2049,17 @@ Cliquez &quot;Suivant&quot; pour créer un fichier de configuration et lancer Octave.&lt;/p&gt;
     <message>
         <location line="-30"/>
         <source>&amp;Help</source>
         <translation>&amp;Aide</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Report Bug</source>
-        <translation>Signaler un bug</translation>
+        <translation>Signaler un bogue</translation>
     </message>
 </context>
 <context>
     <name>news_reader</name>
     <message>
         <location line="-1492"/>
         <source>&lt;html&gt;
 &lt;body&gt;
@@ -2093,27 +2117,27 @@ ou activez la connexion aux nouvelles dans l&apos;onglet &quot;Réseau&quot; du dialogue de préférences d&apos;Octave.
     </message>
 </context>
 <context>
     <name>octave_dock_widget</name>
     <message>
         <location filename="../src/octave-dock-widget.cc" line="+67"/>
         <location line="+201"/>
         <source>Undock widget</source>
-        <translation>Détacher le widget</translation>
+        <translation>Détacher la fenêtre</translation>
     </message>
     <message>
         <location line="-191"/>
         <source>Hide widget</source>
-        <translation>Cacher le widget</translation>
+        <translation>Cacher la fenêtre</translation>
     </message>
     <message>
         <location line="+136"/>
         <source>Dock widget</source>
-        <translation>Attacher le widget</translation>
+        <translation>Attacher la fenêtre</translation>
     </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+237"/>
         <source>Help on</source>
         <translation>Aide sur</translation>
@@ -2121,22 +2145,22 @@ ou activez la connexion aux nouvelles dans l&apos;onglet &quot;Réseau&quot; du dialogue de préférences d&apos;Octave.
     <message>
         <location line="+2"/>
         <source>Documentation on</source>
         <translation>Documentation sur</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Edit</source>
-        <translation>Editer</translation>
+        <translation>Éditer</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>dbstop if ...</source>
-        <translation type="unfinished"></translation>
+        <translation>dbstop si ...</translation>
     </message>
 </context>
 <context>
     <name>octave_qt_link</name>
     <message>
         <location filename="../src/octave-qt-link.cc" line="+136"/>
         <location line="+9"/>
         <location line="+11"/>
@@ -2155,22 +2179,22 @@ n&apos;existe pas. Voulez-vous le créer ?</translation>
     <message>
         <location line="+3"/>
         <source>Octave Editor</source>
         <translation>Éditeur d&apos;Octave</translation>
     </message>
     <message>
         <location line="+254"/>
         <source>The file %1 does not exist in the load path.  To run or debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        <translation>Le fichier %1 ne se trouve pas dans les chemins accessibles.  Pour exécuter ou débuguer la function éditée, vous devez changer de dossier vers %2 ou ajouter ce dossier aux chemins accessibles.</translation>
+        <translation>Le fichier %1 ne se trouve pas dans les chemins accessibles.  Pour exécuter ou déboguer la fonction éditée, vous devez changer de dossier vers %2 ou ajouter ce dossier aux chemins accessibles.</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path. To run or debug the function you are editing, change to the directory %2.</source>
-        <translation>Le fichier %1 est masqué par un fichier du même nom dans les chemins accessibles.  Pour exécuter ou débuguer la fonction éditée, vous devez changer de dossier vers %2.</translation>
+        <translation>Le fichier %1 est masqué par un fichier du même nom dans les chemins accessibles.  Pour exécuter ou déboguer la fonction éditée, vous devez changer de dossier vers %2.</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory or Add Directory to Load Path</source>
         <translation>Changer de répertoire ou ajouter le répertoire aux chemins accessibles</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2238,27 +2262,27 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="-338"/>
         <source>Octave logo only</source>
         <translation>Logo d&apos;Octave seulement</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
-        <translation>Icones textuelles</translation>
+        <translation>Icônes textuelles</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Graphic icons</source>
-        <translation>Icones graphiques</translation>
+        <translation>Icônes graphiques</translation>
     </message>
     <message>
         <location line="-118"/>
         <source>Dock widget title bar</source>
-        <translation>Titre du Dock widget</translation>
+        <translation>Titre de la fenêtre</translation>
     </message>
     <message>
         <location line="+60"/>
         <source>Normal</source>
         <translation>Normal</translation>
     </message>
     <message>
         <location line="+178"/>
@@ -2288,22 +2312,22 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="-69"/>
         <source>Draw a long line marker</source>
         <translation>Montrer le repère de longue ligne</translation>
     </message>
     <message>
         <location line="+155"/>
         <source>Show EOL characters</source>
-        <translation>Montrer les caratères de fin de ligne (EOL)</translation>
+        <translation>Montrer les caractères de fin de ligne (EOL)</translation>
     </message>
     <message>
         <location line="+662"/>
         <source>Default EOL mode</source>
-        <translation>Fin de lingne par défaut (EOL)</translation>
+        <translation>Fin de ligne par défaut (EOL)</translation>
     </message>
     <message>
         <location line="-394"/>
         <source>Indent width</source>
         <translation>Largeur de l&apos;indentation</translation>
     </message>
     <message>
         <location line="+7"/>
@@ -2363,17 +2387,17 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="+574"/>
         <source>Editor Styles</source>
         <translation>Affichage</translation>
     </message>
     <message>
         <location line="+287"/>
         <source>Terminal Colors</source>
-        <translation>Couleurs de la fenetre de commandes</translation>
+        <translation>Couleurs de la fenêtre de commandes</translation>
     </message>
     <message>
         <location line="-173"/>
         <source>Font</source>
         <translation>Police</translation>
     </message>
     <message>
         <location line="-1007"/>
@@ -2383,17 +2407,17 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="+81"/>
         <source>Highlight current line</source>
         <translation>Surligner la ligne courante</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Show complete path in window title</source>
-        <translation>Afficher le chemin complet dans le titre de la fenetre</translation>
+        <translation>Afficher le chemin complet dans le titre de la fenêtre</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Show whitespace</source>
         <translation>Afficher les espaces</translation>
     </message>
     <message>
         <location line="+157"/>
@@ -2403,17 +2427,17 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="+382"/>
         <source>Number of characters before list is shown: </source>
         <translation>Nombre de caractères avant affichage de la liste :</translation>
     </message>
     <message>
         <location line="+124"/>
         <source>Create nonexistent files without prompting</source>
-        <translation>Créer les fichiers non-existants sans prévenir</translation>
+        <translation>Créer les fichiers non existants sans prévenir</translation>
     </message>
     <message>
         <location line="-441"/>
         <source>command line (%f=file, %l=line):</source>
         <translation>ligne de commande (%f=fichier, %l=ligne) :</translation>
     </message>
     <message>
         <location line="+613"/>
@@ -2527,17 +2551,17 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="+64"/>
         <source>max.</source>
         <translation>max.</translation>
     </message>
     <message>
         <location line="-231"/>
         <source>This works well for monospaced fonts. The line is drawn at a position based on the width of a space character in the default font. It may not work very well if styles use proportional fonts or if varied font sizes or bold, italic and normal texts are used.</source>
-        <translation>Ceci fonctionne bien avec les polices typographiques (monospace). La ligne est dessinée tenant compte de la largeur d&apos;un carctère de la police par défaut. Cela peut disfonctionner lors de l&apos;utilisation de polices à tailles variables ou de caractères gras et italiques.</translation>
+        <translation>Ceci fonctionne bien avec les polices typographiques (monospace). La ligne est dessinée tenant compte de la largeur d&apos;un caractère de la police par défaut. Cela peut dysfonctionner lors de l&apos;utilisation de polices à tailles variables ou de caractères gras et italiques.</translation>
     </message>
     <message>
         <location line="+289"/>
         <source>Enable Code Folding</source>
         <translation>Activer le Masquage (repliage) de Code</translation>
     </message>
     <message>
         <location line="+551"/>
@@ -2557,17 +2581,17 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="-532"/>
         <source>Show horizontal scroll bar</source>
         <translation>Montrer la barre de défilement horizontal</translation>
     </message>
     <message>
         <location line="-299"/>
         <source>Show tool bar</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher la barre d&apos;outils</translation>
     </message>
     <message>
         <location line="+383"/>
         <source>Indentation</source>
         <translation>Indentation</translation>
     </message>
     <message>
         <location line="+117"/>
@@ -2602,98 +2626,98 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="+19"/>
         <source>Reload externally changed files without prompt</source>
         <translation>Recharger automatiquement les fichiers modifiés depuis l&apos;extérieur d&apos;Octave</translation>
     </message>
     <message>
         <location line="+93"/>
         <source>Text encoding used for loading and saving</source>
-        <translation type="unfinished"></translation>
+        <translation>Encodage utilisé pour enregistrer et charger</translation>
     </message>
     <message>
         <location line="+73"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as a difference from the default size), font style (&lt;b&gt;b&lt;/b&gt;old, &lt;b&gt;i&lt;/b&gt;talic, &lt;b&gt;u&lt;/b&gt;nderline), text color, and background color (for the latter, the color magenta (255,0,255) is a placeholder for the default background color).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sélectionner la taille de police (difference par rapport à la taille par défaut), son style (&lt;b&gt;g&lt;/b&gt;ras, &lt;b&gt;i&lt;/b&gt;talique, &lt;b&gt;s&lt;/b&gt;ouligné), sa colouleur et la couleur de l&apos;arière-plan (pour ce dernier, la couleur magenta (255,0,255) représente la couleur pas défaut).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sélectionner la taille de police (difference par rapport à la taille par défaut), son style (&lt;b&gt;g&lt;/b&gt;ras, &lt;b&gt;i&lt;/b&gt;talique, &lt;b&gt;s&lt;/b&gt;ouligné), sa couleur et la couleur de l&apos;arrière-plan (pour ce dernier, la couleur magenta (255,0,255) représente la couleur pas défaut).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+192"/>
         <source>(Changing buffer size clears history)</source>
         <translation>(Changer la taille du tampon efface l&apos;historique)</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>History buffer Size</source>
         <translation>Taille du tampon mémoire de l&apos;historique</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>Print debug location in terminal window in addition to the marker in the editor</source>
-        <translation>Afficher la localisation du débuguage dans le terminal en plus du marqueur dans l&apos;éditeur</translation>
+        <translation>Afficher la localisation du déboguage dans le terminal en plus du marqueur dans l&apos;éditeur</translation>
     </message>
     <message>
         <location line="+87"/>
         <source>Behavior</source>
         <translation>Comportement</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Restore last directory of previous session</source>
-        <translation>Revenir au répertoire de la session precédente</translation>
+        <translation>Revenir au répertoire de la session précédente</translation>
     </message>
     <message>
         <location line="+54"/>
         <source>Extensions of files to be opened in the default text editor (separated by &quot;;&quot;):</source>
-        <translation type="unfinished"></translation>
+        <translation>Extensions des fichiers à ouvrir dans l&apos;éditeur de texte (séparées par &quot;;&quot;):</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Workspace</source>
         <translation>Espace de travail</translation>
     </message>
     <message>
         <location line="+35"/>
         <source>Colors for variable attributes</source>
-        <translation type="unfinished"></translation>
+        <translation>Couleurs pour les attributs de variables</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Hide tool tips</source>
         <translation>Masquer les bulles d&apos;aide</translation>
     </message>
     <message>
         <location line="+101"/>
         <source>Import shortcut set</source>
-        <translation type="unfinished"></translation>
+        <translation>Importer un ensemble de raccourcis</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Export current shortcut set</source>
-        <translation type="unfinished"></translation>
+        <translation>Exporter l&apos;ensemble des raccourcis</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Reset shortcuts to their defaults</source>
-        <translation type="unfinished"></translation>
+        <translation>Réinitialiser les raccourcis</translation>
     </message>
     <message>
         <location line="+3"/>
         <location line="+80"/>
         <source>Default</source>
-        <translation type="unfinished">Défaut</translation>
+        <translation>Défaut</translation>
     </message>
     <message>
         <location line="-71"/>
         <source>Edit a shortcut by double-clicking in a cell</source>
         <translation>Modifier un raccourci en double-cliquant sur une cellule</translation>
     </message>
     <message>
         <location line="+76"/>
         <source>Actual</source>
-        <translation type="unfinished"></translation>
+        <translation>Courant</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Network</source>
         <translation>Réseau</translation>
     </message>
     <message>
         <location line="+23"/>
@@ -2708,42 +2732,42 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="-23"/>
         <source>HttpProxy</source>
         <translation>Proxy HTTP</translation>
     </message>
     <message>
         <location line="-2039"/>
         <source>Icon set for dock widgets</source>
-        <translation>Jeu d&apos;icones pour les widgets</translation>
+        <translation>Jeu d&apos;icônes pour les fenêtres</translation>
     </message>
     <message>
         <location line="-48"/>
         <source>Language (requires restart)</source>
         <translation>Langue (nécessite un redémarrage)</translation>
     </message>
     <message>
         <location line="-58"/>
         <source>Icon size</source>
-        <translation>Taille des icones</translation>
+        <translation>Taille des icônes</translation>
     </message>
     <message>
         <location line="+1716"/>
         <source>Synchronize Octave working directory with file browser</source>
         <translation>Synchroniser le répertoire de travail d&apos;Octave et le navigateur de fichiers</translation>
     </message>
     <message>
         <location line="+184"/>
         <source>Shortcuts</source>
         <translation>Raccourcis</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Select this option to prevent conflicts with readline shortcuts</source>
-        <translation>Choisir cette option pour éviter les conflits avec les raccoucis readline</translation>
+        <translation>Choisir cette option pour éviter les conflits avec les raccourcis readline</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Disable global shortcuts when terminal window has focus</source>
         <translation>Désactiver les raccourcis globaux quand la fenêtre de commande est active</translation>
     </message>
     <message>
         <location line="+53"/>
@@ -2763,17 +2787,17 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="+83"/>
         <source>Socks5Proxy</source>
         <translation>Proxy Socks5</translation>
     </message>
     <message>
         <location line="-16"/>
         <source>Hostname:</source>
-        <translation>Nom de l&apos;hote :</translation>
+        <translation>Nom de l&apos;hôte :</translation>
     </message>
     <message>
         <location line="+44"/>
         <source>Proxy type:</source>
         <translation>Type de Proxy :</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -2788,39 +2812,39 @@ Le GUI d&apos;Octave doit maintenant être fermé.</translation>
     <message>
         <location line="+37"/>
         <source>Password:</source>
         <translation>Mot de passe :</translation>
     </message>
     <message>
         <location filename="../src/settings-dialog.cc" line="+127"/>
         <source>Octave Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Préférences Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save settings.  Missing settings file or unknown directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossible de sauvegarder les préférences.  Fichier manquant ou dossier inconnu.</translation>
     </message>
     <message>
         <location line="+18"/>
         <location line="+4"/>
         <location line="+444"/>
         <source>System setting</source>
-        <translation>Reglages système</translation>
+        <translation>Réglages système</translation>
     </message>
     <message>
         <location line="-232"/>
         <source>IBeam Cursor</source>
         <translation>Curseur Trait Vertical</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Block Cursor</source>
-        <translation>Curseur Block</translation>
+        <translation>Curseur Bloc</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
         <translation>Curseur Tiret Bas</translation>
     </message>
     <message>
         <location line="+503"/>
@@ -2949,17 +2973,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Select All</source>
         <translation>Tout Sélectionner</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Clear Clipboard</source>
-        <translation>Effacer le Presse-Papiers</translation>
+        <translation>Effacer le Presse-papiers</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Find in Files</source>
         <translation>Rechercher dans les Fichiers</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2989,17 +3013,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Continue</source>
         <translation>Continuer</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Quit Debug Mode</source>
-        <translation>Quitter le mode Débuguage</translation>
+        <translation>Quitter le mode Déboguage</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show Command Window</source>
         <translation>Afficher la Fenêtre de Commande</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3100,27 +3124,27 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Community News</source>
         <translation>Nouvelles de la Communauté</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Edit Function</source>
-        <translation>Editer la Fonction</translation>
+        <translation>Éditer la Fonction</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Save File</source>
-        <translation>Enregister un Fichier</translation>
+        <translation>Enregistrer un Fichier</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Save File As</source>
-        <translation>Enregister un Fichier Sous</translation>
+        <translation>Enregistrer un Fichier Sous</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Close</source>
         <translation>Fermer</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -3145,22 +3169,22 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Find and Replace</source>
         <translation>Rechercher et Remplacer</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Find Next</source>
-        <translation type="unfinished"></translation>
+        <translation>Trouver le Suivant</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Find Previous</source>
-        <translation type="unfinished"></translation>
+        <translation>Trouver le Précédent</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to Start of Word</source>
         <translation>Effacer jusqu&apos;au Début du Mot</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3277,88 +3301,89 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+4"/>
         <source>Show Line Numbers</source>
         <translation>Afficher les Numéros des Lignes</translation>
     </message>
     <message>
         <location line="+48"/>
         <source>Switch to Left Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>Ouvrir l&apos;Onglet de Gauche</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Switch to Right Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>Ouvrir l&apos;Onglet de Droite</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Move Tab Left</source>
-        <translation type="unfinished"></translation>
+        <translation>Déplacer l&apos;Onglet à Gauche</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Move Tab Right</source>
-        <translation type="unfinished"></translation>
+        <translation>Déplacer l&apos;Onglet à Droite</translation>
     </message>
     <message>
         <location line="+44"/>
         <source>Global</source>
-        <translation type="unfinished"></translation>
+        <translation>Global</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>Tabs</source>
-        <translation type="unfinished"></translation>
+        <translation>Onglets</translation>
     </message>
     <message>
         <location line="+98"/>
         <source>Enter new Shortcut</source>
-        <translation type="unfinished"></translation>
+        <translation>Entrer un nouveau raccourci</translation>
     </message>
     <message>
         <location line="+143"/>
         <source>Overwriting Shortcuts</source>
-        <translation type="unfinished"></translation>
+        <translation>Écrasement des Raccourcis</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>You are about to overwrite all shortcuts.
 Would you like to save the current shortcut set or cancel the action?</source>
-        <translation type="unfinished"></translation>
+        <translation>Vous vous apprêtez à écraser tous le raccourcis.
+Voulez sauvegarder les raccourcis actuels ou annuler cette action ?</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Don&apos;t save</source>
-        <translation type="unfinished"></translation>
+        <translation>Ne pas enregistrer</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Import shortcuts from file ...</source>
-        <translation type="unfinished"></translation>
+        <translation>Importer des raccourcis depuis un fichier ...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Export shortcuts into file ...</source>
-        <translation type="unfinished"></translation>
+        <translation>Exporter les raccourcis vers un fichier ...</translation>
     </message>
     <message>
         <location line="-571"/>
         <source>Step</source>
         <translation>Avancer d&apos;un pas</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Reset Default Window Layout</source>
         <translation>Rétablir la disposition par défaut des fenêtres</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Report Bug</source>
-        <translation>Signaler un bug</translation>
+        <translation>Signaler un bogue</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Close Other Files</source>
         <translation>Fermer les autres fichiers</translation>
     </message>
     <message>
         <location line="+30"/>
@@ -3398,27 +3423,27 @@ Would you like to save the current short
     <message>
         <location line="+2"/>
         <source>Show Long Line Marker</source>
         <translation>Afficher le Repère de Longue Ligne</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Toolbar</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher la barre d&apos;outils</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Statusbar</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher la barre de statut</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Horizontal Scrollbar</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher la barre de navigation horizontale</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom In</source>
         <translation>Agrandir</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -3454,17 +3479,17 @@ Would you like to save the current short
     <message>
         <location line="+4"/>
         <source>Run File</source>
         <translation>Exécuter le Script</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Run Selection</source>
-        <translation>Exécuter la Selection</translation>
+        <translation>Exécuter la Sélection</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Help on Keyword</source>
         <translation>Aide sur le Mot de Clé</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3482,17 +3507,17 @@ Would you like to save the current short
         <location line="+23"/>
         <source>Edit</source>
         <translation>Éditer</translation>
     </message>
     <message>
         <location line="-21"/>
         <location line="+25"/>
         <source>Debug</source>
-        <translation>Débuguer</translation>
+        <translation>Déboguer</translation>
     </message>
     <message>
         <location line="-23"/>
         <source>Window</source>
         <translation>Fenêtre</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3683,47 +3708,47 @@ voulez l&apos;affecter à la présente action et le retirer de la précédente ?</translation>
     <message>
         <location line="+3"/>
         <source>Rename</source>
         <translation>Renommer</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Only top-level symbols may be renamed</source>
-        <translation type="unfinished"></translation>
+        <translation>Seuls les symboles du plus haut niveau peuvent être renommés</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>Hide filter</source>
-        <translation type="unfinished"></translation>
+        <translation>Cacher le filtre</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show filter</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher le filtre</translation>
     </message>
     <message>
         <location line="+149"/>
         <source>Class</source>
-        <translation type="unfinished">Classe</translation>
+        <translation>Classe</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Dimension</source>
-        <translation type="unfinished">Dimensions</translation>
+        <translation>Dimensions</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Value</source>
-        <translation type="unfinished">Valeur</translation>
+        <translation>Valeur</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Attribute</source>
-        <translation type="unfinished">Attribut</translation>
+        <translation>Attribut</translation>
     </message>
     <message>
         <location line="-17"/>
         <source>View the variables in the active workspace.&lt;br&gt;</source>
         <translation>Voir les variables dans l&apos;espace de travail.&lt;br&gt;</translation>
     </message>
     <message>
         <location line="+1"/>
